scout/server/blueprints/institutes/views.py

```diff
@@ -193,7 +193,7 @@ def gene_variants(institute_id):
             )
 
         data = controllers.gene_variants(
-            store, results, result_size, page
+            store, results=results, page=page
```

Comment:
Unnamed arguments is a pain, quod erat demonstrandum.

Reply:
I know!